src/Proyecto26.RestClient/Helpers/HttpBase.cs

```diff
@@ -8,6 +8,8 @@ namespace Proyecto26
 {
     public static class HttpBase
     {
+        public static int HTTP_NO_CONTENT = 204;
```

Comment:
This should probably be a `const` instead of `static`. Might be good to scope it `private` instead of `public` too, since it doesn't seem like anything outside of `HttpBase` will need to know about it.

Reply:
@L-Naej hello mate, thanks for your contribution! Please check the @StevenGarberg's comment and let us know! 👍

Reply:
Let me change that from develop branch, thanks for your help @StevenGarberg! <3

Reply:
You're welcome! Thank you for making such a great library @jdnichollsc. I use it in all of my Unity projects to talk to my web APIs.

Reply:
@StevenGarberg do you want to be a maintainer of this lib? <3

Reply:
@jdnichollsc Absolutely I do!

Reply:
Sorry guys, was completely off from internet for personal reasons, thank you very much for handling this!
Indeed I don't know what I was thinking with these public static, silly mistake...

Reply:
haha no problem, thanks for your help! <3